### DISCUSSION TITLE
config.json file isn't found upon .bat execution

### DISCUSSION THREAD

**Discussion by TheRealChrisWilliams** [TobiasPankner/Teams-Auto-Joiner#152] [Q&A] [answered]
So I tried to automate the process since I can't have my computer checking the site the whole night. Instead, I made it so that it automatically executes auto_joiner.py an hour before the meeting. However, upon execution of the .bat file, I get an error saying that the config.json file isn't found. 

Therefore, aut_joiner.py isn't able to run. Any ideas on how to fix this?

P.S: I know there is a run_at_time field in the .json file, I'd just like to know if it's possible to get the script itself to run at a specified time. 

![1](https://user-images.githubusercontent.com/41800074/137973856-ccbe38a6-d9f3-4019-8323-3af1f2883376.PNG)

**Comment by TobiasPankner** [comment, marked as answer]
If you  want to use a bat to start it, change the directory to the location of the bot before running the python command.

**Reply by TheRealChrisWilliams** [reply]
Yup, changing the directory worked perfectly! Thank you, your code is solid. I'm just trying new things with automation which is why I wanted to use the .bat over the run_at_time field. Thanks again!

**Reply by TobiasPankner** [reply]
You'e welcome!
